src/components/common/ChatInputEmojiPicker.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useState } from 'react';
+import EmojiPicker, { EmojiClickData, EmojiStyle, SkinTones } from 'emoji-picker-react';
+import './ChatInputEmojiPicker.css';
+
+interface ChatInputEmojiPickerProps {
+	onEmojiClick: (emojiData: EmojiClickData, e: MouseEvent) => void;
+}
+export default function ChatInputEmojiPicker(props: ChatInputEmojiPickerProps): JSX.Element {
+	const BUTTON_HINT = 'Pick an emoji';
```

Comment:
Please move this constant to localization object.

Reply:
Done